docs/src/man/getting_started.md

```diff
@@ -471,3 +471,70 @@ CSV.write(output, df)
 ```
 
 The behavior of CSV functions can be adapted via keyword arguments. For more information, see `?CSV.read` and `?CSV.write`, or checkout the online [CSV.jl documentation](https://juliadata.github.io/CSV.jl/stable/).
+
+## Broadcasting
```

Comment:
My point with this PR is exactly to decide what kind of broadcasting we want to provide by default. We essentially have four options for broadcasting purposes:
* make `DataFrame` be treated as scalar, then we do `Ref(df)`;
* make `DataFrame` be treated as a collection of rows, then we do `eachrow(df)`;
* make `DataFrame` be treated as a collection of rows, then we do `eachcol(df, false)`;
* make `DataFrame` be treated as a two dimensional object, like array; an inefficient way to do this would be `Matrix(df)`, probably a more efficient implementation would be needed if we decided on it.

We have to select one. The other three have to be keyed-in by the user manually when broadcasting.

So the question is - how do we want to treat an `AbstractDataFrame` in broadcasting by default (when it is not wrapped by some other object).